en/hubs_tos.md

```diff
@@ -0,0 +1,82 @@
+# Terms of Service for Hubs
+
+Version 1.0, Effective October 11, 2022
+{: datetime="2022-10-11" }
+
+Hubs (the “Service”) is a new, preview service that allows you to set up and manage your own set of real-time communications platforms for virtual reality, augmented reality, desktop, laptop, or mobile. It is available as a paid subscription service.  
+
+These Terms of Service explain your rights and responsibilities when you use Hubs. By purchasing and using Hubs, you assure Mozilla that you have read these terms and agree to them. 
+
+## 1. Subscribing to Hubs
+
+### You’ll Need A Firefox Account
+A Firefox Account is required to use the Service. To create a Firefox Account, you will also need to agree to the [Terms of Service](https://www.mozilla.org/about/legal/terms/services/) and [Privacy Notice](https://www.mozilla.org/privacy/firefox/) for your Firefox Account.
+
+### Your Privacy
+__Privacy.__ You are responsible for the privacy of the individuals who use your Hubs. This includes compliance with all applicable privacy laws. The [Hubs Privacy Notice](https://www.mozilla.org/privacy/hubs-subscription/) explains how Mozilla receives and handles data to operate the Service.
+
+Mozilla is a global organization, and our computers and service providers’ computers may be in various countries around the world, including the United States. This means that your information and your users’ information might be processed on servers located outside of the country where you and your users live, and that country may have a different level of data protection regulation than yours. By using the Service, you assure Mozilla that you consent to this kind of transfer of users’ information and that you have obtained necessary consent from the individuals who use your Hubs.
+Your Payment
```

Comment:
Should this be a subheading?

Reply:
I changed this around to address a duplicate heading.